venia-integration-tests/src/fixtures/wishlist/index.js

```diff
@@ -1,6 +1,12 @@
-export const wishistRoute = './wishlist';
+export const wishistRoute = 'wishlist';
 
 export const wishlistRemove = 'Remove';
 
 export const removeProductMessage =
     'Are you sure you want to delete this product from the list?';
+
+export const singleWishlistHeading = 'Remove';
+
+export const multipleWishlistHeading = 'Remove';
+
+export const emptyWishlistMessage = 'There are currently no items in this list';
```

Comment:
These will likely get out of sync as Venia is updated over time but as long as these tests are run (and updated as needed) as part of the QA process I think we can keep them synced up and passing.